alexandria/core/filters.py

```diff
@@ -168,10 +168,21 @@ class FileFilterSet(FilterSet):
     document_metainfo = JSONValueFilter(field_name="document__metainfo")
     active_group = ActiveGroupFilter()
     files = BaseCSVFilter(field_name="pk", lookup_expr="in")
+    category = CategoriesFilter(field_name="document__category")
+    categories = CategoriesFilter(field_name="document__category")
```

Comment:
Do we really need both? They're functionally equal. I'd rather only have the plural if possible. (I notice the `DocumentFilterSet` also has both though, so maybe for consistency... :thinking: )